lib/plugins/cpuUsageThrottle.js

```diff
@@ -86,39 +86,41 @@ function cpuUsageThrottlePlugin (opts) {
     assert.optionalNumber(opts.interval, 'opts.interval');
     assert.optionalNumber(opts.halfLife, 'opts.halfLife');
 
-    var self = {};
-    self._limit = (typeof opts.limit === 'number') ?
-      opts.limit : 0.75;
-    self._max = opts.max || 1;
-    self._interval = opts.interval || 250;
-    self._halfLife = (typeof opts.halfLife === 'number') ? opts.halfLife : 250;
-    assert.ok(self._max > self._limit, 'limit must be less than max');
+    var plugin = {};
+    plugin._limit = (typeof opts.limit === 'number') ?
+        opts.limit : 0.75;
+    plugin._max = opts.max || 1;
+    plugin._interval = opts.interval || 250;
+    plugin._halfLife = (typeof opts.halfLife === 'number') ? opts.halfLife :
+        250;
+    assert.ok(plugin._max > plugin._limit, 'limit must be less than max');
 
-    self._ewma = new EWMA(self._halfLife);
+    plugin._ewma = new EWMA(plugin._halfLife);
 
-    // self._reject represents the % of traffic that we should reject at the
+    // plugin._reject represents the % of traffic that we should reject at the
     // current point in time based on how much over our limit we are. This is
     // updated on an interval by updateReject().
-    self._reject = 0;
+    plugin._reject = 0;
 
-    // self._timeout keeps track of the current handle for the setTimeout we
+    // plugin._timeout keeps track of the current handle for the setTimeout we
     // use to gather CPU load averages, this allows us to cancel the timeout
     // when shutting down restify.
-    self._timeout = null;
```

Comment:
Why is it changing all the `self`s to `plugin`?

Reply:
`consistent-this` rule forces to use `self` for `this`

Reply:
Self is not the `this` here.